File: PRAK110-2310817210004-Allano Lintang Ertantora.py
import math
alas = 5
tinggi = 12
hyp = round(math.sqrt((alas**2)+(tinggi**2)))#rumus pythagoras untuk mencari sisi miring
print("Diketahui :")
print("Alas =",alas,"cm")
print("Tinggi =",tinggi,"cm")
for i in range(2):
    print(" ")
print("Jawab:")
print("Sisi A =", tinggi,"cm")
print("Sisi B =", hyp,"cm")
print("Sisi C =", alas,"cm")
print("Keliling =", tinggi + alas + hyp)
print("Luas =", round((1/2) * alas * tinggi))

File: Praktikum 2/C/PRAK110-2310817210004-Allano-Lintang-Ertantora.c
#include <stdio.h>
#include <math.h>

int main() {
    int alas = 5;
    int tinggi = 12;
    int hyp = sqrt(pow(alas, 2) + pow(tinggi, 2)); //rumus pythagoras untuk mencari sisi miring
    printf("Diketahui :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n\n\n", tinggi);
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", tinggi);
    printf("Sisi B = %d cm\n", hyp);
    printf("Sisi C = %d cm\n", alas);
    printf("Keliling = %d cm\n", alas + tinggi + hyp);
    printf("Luas = %d cm", (alas * tinggi)/2);
    return 0;
}